redash/models/changes.py

```diff
@@ -0,0 +1,86 @@
+from sqlalchemy.inspection import inspect
+from sqlalchemy_utils.models import generic_repr
+
+from .base import GFKBase, db, Column
+from .types import PseudoJSON
+
+
+@generic_repr('id', 'object_type', 'object_id', 'created_at')
+class Change(GFKBase, db.Model):
+    id = Column(db.Integer, primary_key=True)
+    # 'object' defined in GFKBase
+    object_version = Column(db.Integer, default=0)
+    user_id = Column(db.Integer, db.ForeignKey("users.id"))
+    user = db.relationship("User", backref='changes')
+    change = Column(PseudoJSON)
+    created_at = Column(db.DateTime(True), default=db.func.now())
+
+    __tablename__ = 'changes'
+
+    def to_dict(self, full=True):
+        d = {
+            'id': self.id,
+            'object_id': self.object_id,
+            'object_type': self.object_type,
+            'change_type': self.change_type,
```

Comment:
`self.change_type` doesn't exist and I think it causes issues in `tests/handlers/test_queries.py`